umh-core/pkg/config/redpandaserviceconfig/comparator.go

```diff
@@ -66,6 +66,11 @@ func (c *Comparator) ConfigDiff(desired, observed RedpandaServiceConfig) string
 			normDesired.Topic.DefaultTopicRetentionBytes, normObserved.Topic.DefaultTopicRetentionBytes))
 	}
 
+	if normDesired.Topic.DefaultTopicCompressionType != normObserved.Topic.DefaultTopicCompressionType {
```

Comment:
this results in a breaking change right?

old configs will not have snappy, so this will always fail, but the system can also not reconcile towards the new state as the updating does not work.

so it will keep failing.

Reply:
i would like to prevent issues like this so that updating does not cause a bricked instance.